packages/ui/Toggleable/Toggleable.js

```diff
@@ -231,21 +247,38 @@ const ToggleableHOC = hoc(defaultConfig, (config, Wrapped) => {
 			() => this.updateActive(!this.state.active)
 		)
 
+		getCustomAriaProps () {
+			const label = this.props[this.state.active ? 'toggleOnAriaLabel' : 'toggleOffAriaLabel'];
+
+			// You can override aria-valuetext and role property on the toggleable component.
+			return label ? {'aria-valuetext': label, role: 'spinbutton'} : null;
```

Comment:
We are using `aria-pressed` prop to manage select state in the toggle button. You are right `spinbutton` role looks not accurate but this seems the only way to read custom string for reading without `pressed` status of the web engine - "on / off" and only read the app developers custon string mentioned in https://github.com/enyojs/enact-internal-samples/pull/56#discussion_r229532406. 
Hm.. what do you think? shall we defer for finding out better solution? This requirement seems not urgent, internally request

Reply:
Adding my opinion. 
I propose to apply `aria-label` with the  toggleOffAriaLabel and toggleOnAriaLabel value rather than `aria-valuetext` with `spinbutton`. 
Then app developer easily apply aria string of the toggle button but web engine always reads pressed status(on/off) after reading button aria string. 
But this makes sense to me. If we remove to read pressed status by webengine("on/off"), would it be confused to know this is the toggle button that user(Blind) can toggle?

Reply:
I think that if a component is selectable and that state is represented by an ARIA attribute, it makes sense for the TTS to handle it consistently and not force an alternate behavior from the framework. If an app wanted to do so, they could override `role` for their instance to be something else (and maybe add `aria-live` too) to achieve the desired result but I'm not convinced it's an appropriate change.

Should we get the TTS/accessibility folks to weigh in on this more?

Reply:
I agree. I also don't like to break the semantic for aria attribute in the framework level. 
No, I don't think we should talk with TTS engineer for this at this moment. We don't have TV's specific requirement yet. We can just keep as-is